sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -1316,6 +1317,34 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
     }
   }
 
+  test("SPARK-39393 Do not push down predicate filters for repeated primitive fields") {
```

Comment:
Nice work. Little suggestion: 
```suggestion
  test("SPARK-39393: Do not push down predicate filters for repeated primitive fields") {
```

Reply:
Thanks. Done.